lcmgen/lcmgen.c

```diff
@@ -694,13 +695,8 @@ lcm_struct_t *parse_struct(lcmgen_t *lcmgen, const char *lcmfile, tokenize_t *t)
 
     parse_require(t, "{");
 
-    while (!parse_try_consume(t, "}")) {
-        // Check for leading comments that will be used to document the member.
-        parse_try_consume_comment(lcmgen, t, 1);
-
-        if (parse_try_consume(t, "}")) {
-            break;
-        }
+    // Store any leading comments that will be used to document the member.
+    while (!parse_try_consume(lcmgen, t, "}", TRUE)) {
```

Comment:
It seems that this argument is always `FALSE`, except for right here. So might we have keep the comment parsing in this area and done without the extra argument?